tests/realtikvtest/testkit.go

```diff
@@ -53,6 +53,7 @@ func RunTestMain(m *testing.M) {
 	})
 	tikv.EnableFailpoints()
 	opts := []goleak.Option{
+		goleak.IgnoreTopFunction("github.com/pingcap/goleveldb/leveldb.(*DB).mpoolDrain"),
```

Comment:
Since we wait for goleveldb closed below, perhaps we don't need this line.

Reply:
removed it.